src/ray/raylet/raylet_util.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <csignal>
+
+namespace ray {
+namespace raylet {
+
+inline void ShutdownRayletGracefully() {
```

Comment:
didn';t review thinking it is copy/paste

Reply:
That's true.